Index: widelands-build20/CMakeLists.txt
===================================================================
--- widelands-build20/CMakeLists.txt
+++ widelands-build20/CMakeLists.txt	2019-05-06 11:39:31.599373673 +0200
@@ -162,7 +162,7 @@
     set(WL_OPTIMIZE_FLAGS "-O3")
   endif ()
   set(WL_DEBUG_FLAGS "-DNDEBUG -DNOPARACHUTE")
-  option(OPTION_ASAN "Build with AddressSanitizer" ON)
+  option(OPTION_ASAN "Build with AddressSanitizer" OFF)
 else()
   message(FATAL_ERROR "Unknown CMAKE_BUILD_TYPE: ${CMAKE_BUILD_TYPE}")
 endif()
@@ -313,7 +313,7 @@
 install (
   FILES ${CMAKE_CURRENT_BINARY_DIR}/VERSION
   DESTINATION ${WL_INSTALL_BASEDIR}
-  CONFIGURATIONS Debug;Release
+  CONFIGURATIONS Debug;Release;RelWithDebInfo
   COMPONENT CoreVersionFile
 )
 
@@ -338,7 +338,7 @@
     data/txts
     data/world
   DESTINATION ${WL_INSTALL_DATADIR}
-  CONFIGURATIONS Debug;Release
+  CONFIGURATIONS Debug;Release;RelWithDebInfo
   COMPONENT CoreDataFiles
 )
 
@@ -346,7 +346,7 @@
   DIRECTORY
     data/maps
   DESTINATION ${WL_INSTALL_DATADIR}
-  CONFIGURATIONS Debug;Release
+  CONFIGURATIONS Debug;Release;RelWithDebInfo
   COMPONENT MapFiles
 )
 
@@ -355,7 +355,7 @@
     data/music
     data/sound
   DESTINATION ${WL_INSTALL_DATADIR}
-  CONFIGURATIONS Debug;Release
+  CONFIGURATIONS Debug;Release;RelWithDebInfo
   COMPONENT MusicFiles
 )
 
@@ -365,7 +365,7 @@
     CREDITS
     ChangeLog
   DESTINATION ${WL_INSTALL_BASEDIR}
-  CONFIGURATIONS Debug;Release
+  CONFIGURATIONS Debug;Release;RelWithDebInfo
   COMPONENT CoreLicenseFiles
 )
 
@@ -384,6 +384,6 @@
   DIRECTORY
     ${CMAKE_CURRENT_BINARY_DIR}/locale/
   DESTINATION ${WL_INSTALL_DATADIR}/locale
-  CONFIGURATIONS Debug;Release
+  CONFIGURATIONS Debug;Release;RelWithDebInfo
   COMPONENT CoreLanguageFiles
 )
