Index: widelands-17/src/network/internet_gaming.cc
===================================================================
--- widelands-17.orig/src/network/internet_gaming.cc	2012-04-23 11:46:49.000000000 +0200
+++ widelands-17/src/network/internet_gaming.cc	2012-05-23 15:31:12.000000000 +0200
@@ -362,7 +362,7 @@
 			logout();
 			setError();
 			throw warning
-				(_
+				(_(""), _
 				 	("Expected a LOGIN, RELOGIN or REJECTED packet from server, but received command "
 				 	 "%s. Maybe the metaserver is using a different protocol version ?"),
 				 cmd.c_str());
@@ -403,7 +403,7 @@
 			std::string type     = packet.String();
 
 			if (type != "public" && type != "private" && type != "system")
-				throw warning("Invalid chat message type \"%s\".", type.c_str());
+				throw warning("","Invalid chat message type \"%s\".", type.c_str());
 
 			bool        personal = type == "private";
 			bool        system   = type == "system";
@@ -708,7 +708,7 @@
  */
 bool InternetGaming::str2bool(std::string str) {
 	if ((str != "true") && (str != "false"))
-		throw warning("Conversion from std::string to bool failed. String was \"%s\"", str.c_str());
+		throw warning("","Conversion from std::string to bool failed. String was \"%s\"", str.c_str());
 	return str == "true";
 }
 
Index: widelands-17/src/io/dedicated_log.cc
===================================================================
--- widelands-17.orig/src/io/dedicated_log.cc	2012-04-23 11:46:49.000000000 +0200
+++ widelands-17/src/io/dedicated_log.cc	2012-05-23 15:31:55.000000000 +0200
@@ -78,7 +78,7 @@
 	temp += c.sender.empty() ? "SYSTEM" : c.sender;
 	temp += "</td><td class=\"recipient\"> ->" + c.recipient + "</td><td class=\"message\">";
 	temp += c.msg + "</td></tr>\n";
-	m_chat.Printf(temp.c_str());
+	m_chat.Printf("%s",temp.c_str());
 	m_chat.WriteAppend(*root, m_chat_file_path.c_str());
 }
 
@@ -185,7 +185,7 @@
 		}
 	}
 	temp += "</table>\n";
-	m_chat.Printf(temp.c_str());
+	m_chat.Printf("%s",temp.c_str());
 	m_chat.Write(*root, m_info_file_path.c_str());
 }
 
@@ -197,7 +197,7 @@
 	std::string temp("<tr><td class=\"log\">");
 	temp += msg;
 	temp += "</td></tr>\n";
-	m_chat.Printf(temp.c_str());
+	m_chat.Printf("%s",temp.c_str());
 	m_chat.WriteAppend(*root, m_log_file_path.c_str());
 }
 
