diff -up widelands-build19-src-gcc7/src/graphic/text/bidi.h~ widelands-build19-src-gcc7/src/graphic/text/bidi.h
--- widelands-build19-src-gcc7/src/graphic/text/bidi.h~	2017-06-20 10:33:11.000000000 +0200
+++ widelands-build19-src-gcc7/src/graphic/text/bidi.h	2018-08-14 16:26:52.370288814 +0200
@@ -40,7 +40,7 @@ bool has_rtl_character(std::vector<std::
 // True if a string contains a character from the script's code blocks
 bool has_script_character(const char* input, UI::FontSets::Selector script);
 UI::FontSet const* find_fontset(const char* word, const UI::FontSets& fontsets);
-std::string icustring2string(const UnicodeString& convertme);
+std::string icustring2string(const icu::UnicodeString& convertme);
 std::string icuchar2string(const UChar& convertme);
 bool cannot_start_line(const UChar& c);
 bool cannot_end_line(const UChar& c);
